### ISSUE TITLE
Config to ignore specific 3rd party scripts injected by GTM and run them on the main thread.

### ISSUE THREAD

**Issue by garethweaver** [BuilderIO/partytown#161]
While implementing PartyTown with many scripts injected by GTM we're finding some scripts to be currently incompatible which is blocking us from releasing our update. Would it be possible to configure PartTown to not inject specific tags coming from GTM as `type="text/partytown"` and instead just allow them to be run on the main thread?

This would enable us to pick and choose which 3rd party scripts run in the web-worker and which ones don't. When the issues are resolved with these harder to integrate scripts we can then remove them from the ignore list.

**Comment by steve8708** [comment]
ah yes maybe we could have a config for this, kinda like `resolveUrl`

```html
partytown = {
  loadWithPartytown: (url) => {
     if (/* your own logic for what to load in partytown vs not */) return true;
     return false;
  }
}
```

open to contributions!

**Comment by DanBeckDev** [comment]
Hi everyone, 
I have been actively working on this and currently have a working solution.
If I could be assigned to this it would be great. 
I just need to write tests then I will raise the PR.

**Comment by garethweaver** [comment]
Update: resolved in #200 

Docs here:
https://github.com/BuilderIO/partytown/commit/aefb88b6646a9725bb9da7a778aedd4364b9da59
